src/cdk/schematics/update-tool/utils/line-mappings.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
```

Comment:
Could you add a file-level comment that explains the high-level idea behind this line mapping?

Reply:
Done.